cobalt-transcribe/cmd/models.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (2019) Cobalt Speech and Language Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cobaltspeech/examples-go/cobalt-transcribe/internal/client"
+	"github.com/spf13/cobra"
+)
+
+var listModelsCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List models available in transcribe server.",
+	Args:  addGlobalFlagsCheck(cobra.NoArgs),
+	Run: runClientFunc(func(c *client.Client, args []string) error {
+		ctx, cancelCtx := context.WithCancel(context.Background())
+		defer cancelCtx()
```

Comment:
what I mean by [this](https://github.com/cobaltspeech/examples-go/pull/18#discussion_r1133458446) is
```suggestion
	Run: runClientFunc(func(ctx context.Context, c *client.Client, args []string) error {
```

Reply:
but maybe we're trying to generalize thing too much and that the problem -- current simple client code is not simple and easy to follow.